### ISSUE TITLE
Staging Review finding: Title tags incorrect

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#118763]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** vfs-mhv-secure-messaging
**Product:** mhv-secure-messaging
**Feature:** Oracle Health curated list

## Findings details
**VA.gov Experience Standard - issue:** [User encounters content that doesn't match other content on VA.gov or VA Mobile App.](https://design.va.gov/about/experience-standards/#user-encounters-content-that-is-similar-to-other-content-on-vagov)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @erinrwhite (IA), @jasonday (Accessibility), @kkitagawa-bosch (content)

### Description

On the "Select care team" and "Only use messages for non-urgent needs" pages, the title tag doesn't update to reflect what page the user is on. Instead, it says "Messages: Inbox."

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="793" height="390" alt="Image" src="https://github.com/user-attachments/assets/0952e91b-b678-405a-8975-b12dada203cd" />
<img width="575" height="238" alt="Image" src="https://github.com/user-attachments/assets/091bc365-4558-43a7-b173-411813ff804d" />

</details>

## Recommended action

Update title tags to match the H1s of the pages.

## References

- **Accessibility Defect Severity:** N/A
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** [Title tags](https://design.va.gov/content-style-guide/title-tags)
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by steveg-IAT** [comment]
@rretamoza-va I see this is in your "In Progress" swim lane. Has someone from the SM team picked this up? There is no assignment.
